### PR TITLE
Changed readme to markdown

### PR DESCRIPTION
**Pull request by motyar**

### CODE REVIEW

File: readme.html [hunk 1]
```diff
@@ -1,98 +0,0 @@
...
-</ul>
-
-<h2>Share the Love</h2>
-<p>WordPress has no multi-million dollar marketing campaign or celebrity sponsors, but we do have something even better&#8212;you. If you enjoy WordPress please consider telling a friend, setting it up for someone less knowledgable than yourself, or writing the author of a media article that overlooks us.</p>
-
-<p>WordPress is the official continuation of <a href="http://cafelog.com/">b2/caf&#233;log</a>, which came from Michel V. The work has been continued by the <a href="https://wordpress.org/about/">WordPress developers</a>. If you would like to support WordPress, please consider <a href="https://wordpress.org/donate/" title="Donate to WordPress">donating</a>.</p>
-
-<h2>License</h2>
-<p>WordPress is free software, and is released under the terms of the <abbr title="GNU General Public License">GPL</abbr> version 2 or (at your option) any later version. See <a href="license.txt">license.txt</a>.</p>
```
Comment (IfeanyiEkperi) [inline readme.html]: WordPress is 'a' free software

A is missing in the above statement

Comment (motyar) [reply readme.html]: I havn't changed that, you can fork and make this change on.
check it was same in the HTML file 
https://github.com/WordPress/WordPress/blob/master/readme.html#L95

Comment (IfeanyiEkperi) [reply readme.html]: OK. I will do that now.

**Comment by Nolwennig** [comment]
This repository is just a mirror of the WordPress subversion repository. 

Please do not send pull requests. 

Submit patches to https://core.trac.wordpress.org/ instead.

**Pull request closed without merging**